alpha-blog/db/schema.rb

```diff
@@ -20,8 +20,8 @@
   end
 
   create_table "articles", force: :cascade do |t|
-    t.string "title"
```

Comment:
Where did all these changes come from?

Reply:
I think its from this [PR](https://github.com/LeeRenJie/Learn-Ruby/pull/15/files) but no idea why the update in migration files were not reflected on the schema. Pretty sure I ran `rails db:migrate` a couple of times after that. But just now it appeared after I ran `rails db:migrate` to run and see the app again.